<launch>

<!-- action interface to switch controllers -->
  <node pkg="mechanism_control" type="action_mechanism_control" output="screen" />

<!-- Laser tilt controller -->
  <param name="laser_tilt_controller/autostart" type="bool" value="true" />
  <node pkg="mechanism_control" type="spawner.py" args="$(find pr2_experimental_controllers)/laser_tilt/laser_tilt_controller.xml" />

<!-- Head controller -->
  <param name="head_controller/autostart" type="bool" value="true" />
  <node pkg="mechanism_control" type="spawner.py" args="$(find pr2_default_controllers)/head_pan_tilt_controller.xml" />

<!-- Gripper controller -->
  <param name="gripper_effort/autostart" type="bool" value="false" />
  <node pkg="mechanism_control" type="spawner.py" args="$(find door_handle_detector)/launch/gripper_effort_right.xml" />

</launch>
